File: Week 0 - C/mario-more.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height : ");
    }
    while (height < 1 || height > 8);

    int row;
    for (row = 1; row <= height; row++) // printing rows
    {
        int space;
        for (space = 0; space < height - row; space++) // print space
        {
            printf(" ");
        }

        int column;
        for (column = 1; column <= row; column++) // print left side
        {
            printf("#");
        }

        printf("  "); // print gap

        for (column = 1; column <= row; column++) // print right side
        {
            printf("#");
        }
        printf("\n"); // bringing cursor to new line
    }
}

File: Week 6 - Python/sentimental-mario-more/mario-more.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

for i in range(n):
    for j in range(n + i + 3):
        if (j == n or j == n + 1 or i + j < n - 1):
            print(" ", end="")
        else:
            print("#", end="")
    print()
